website/docs/guides/dbt-python-bigframes.md

```diff
@@ -82,8 +90,16 @@ The dbt BigFrames submission method supports both service account and OAuth cred
    #Grant Colab Entperprise User
    gcloud projects add-iam-policy-binding ${GOOGLE_CLOUD_PROJECT} --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/aiplatform.colabEnterpriseUser
    ```
+   
+   :::info When using a Shared VPC
+   When using Colab Enterprise in a Shared VPC environment, additional roles are required for the following service accounts on the Shared VPC host project:
+   - Vertex AI P4SA (`service-<PROJECT_NUMBER>@gcp-sa-aiplatform.iam.gserviceaccount.com`): This service account always requires the Compute Network User (`roles/compute.networkUser`) role on the Shared VPC host
+     project. Replace `<PROJECT_NUMBER>` with the project number.
```

Comment:
For formatting consistency can we make it so 97 and 99 are on the same lines as the bulleted content?